src/test/java/teammates/test/cases/action/InstructorCourseDeleteAllRecoveryCoursesActionTest.java

```diff
@@ -51,12 +52,43 @@ public void testExecuteAndPostProcess() throws Exception {
         assertEquals(instructor2OfCourse3.courseId, courseList.get(0).getId());
         assertTrue(CoursesLogic.inst().isCoursePresent(instructor2OfCourse3.courseId));
 
-        ______TS("Typical case, delete all courses from Recycle Bin, with privilege");
+        ______TS("Typical case, delete all courses from Recycling Bin, with privilege for only some courses");
```

Comment:
Minor nit, but why did it become `Recycling Bin`? The rest are still `Recycle Bin` as well

Reply:
Sorry... just copy & paste your suggested case name in your comment above without doudle checking :-p